src/Resources/views/storefront/component/listing/filter/filter-multi-select.html.twig

```diff
@@ -1,16 +1,20 @@
 {% sw_extends '@Storefront/storefront/component/listing/filter/filter-multi-select.html.twig' %}
 
 {% block component_filter_multi_select %}
-    <div class="{% if hidden %}d-none {% endif %}filter-multi-select filter-multi-select-{{ name }} filter-panel-item{% if not sidebar %} dropdown{% endif %}"
-         data-{{ pluginSelector }}="true"
-         data-{{ pluginSelector }}-options='{{ dataPluginSelectorOptions|json_encode }}'>
+    {% if not context.context.extensions.findologicService.enabled %}
```

Comment:
Does enabled stand for navigation?

Reply:
The `FindologicService` is aware of the state and tells us if Findologic should be used or not.